recipes/ocp/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "ocp" %}
+{% set version = "7.5.2beta" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  # The release .tar.gz does not contain the requisite `pywrap` submodule, use 
+  # the tagged git version for now
+
+  # url: "https://github.com/CadQuery/OCP/archive/refs/tags/7.5.2-beta.tar.gz"
+  # sha256: 616303149564dae4628b7f536ccfdd245cf397eaf8b3c1b2f724d517c99ac171
+
+  git_url: https://github.com/CadQuery/OCP.git
+  git_rev: 7.5.2-beta
+
+build:
+  number: 0
+
+test:
+  imports:
+    - OCP
+
+outputs:
+  - name: ocp-devel
+    script: generate-bindings.sh
+    requirements:
+      host:
+        - python
+        - clangdev
+        - clang =10.0.1
+        - pybind11
+        - pybind11-abi
+        - occt =7.5.2
+        # binder requirements
+        - joblib
+        - toml
+        - click
+        - cymbal
+        - jinja2
+        - logzero
+        - pandas
+        - path.py
+        - pyparsing
+        - schema
+        - tqdm
+        - toposort
+
+  - name: {{ name }}
+    script: compile-bindings.sh
+    requirements:
+      build:
+        - python
+        - cmake
+        - ninja
+        - compilers('cxx')
```

Comment:
_fairly_ this should be, e.g. `- {{ compiler("cxx") }}`, though maybe i just don't know enough about that part of the stack